en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,417 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a **cache** or a **database**, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperlogs and geospatial indexes with radius queries.
+
+This connector enables developers to use an external Redis server as a cache or a database in the WSO2 EI mediation logic.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Redis Connector and access the Redis server, using Redis commands.
+
+The user sends the request to invoke an API to get stock details. This REST call will get converted into a SOAP message and is sent to the back-end service. While the response from the backend service is converted back to JSON and sent back to the API caller, WSO2 EI will extract stock volume details from the response and store it into a configured Redis server.
+When users need to retrieve stock volumes collected, they can invoke the `getstockvolumedetails` resource. This example also demonstrates how users can manipulate this stock volume details by removing unwanted items from the Redis server.
+
+> **Note**: In this scenario you need to set up the Redis Server in your local machine. Please refer the [Setting up the Redis Connector](redis-connector-configuration.md) documentation. Follow the steps listed under `Setting up the Redis server` section to setup the Redis server and `Set up the back-end service` section to setup the Stockquote service. 
+This example demonstrates how to use Redis connector to:
+
+1. Retrieve stock volume details from the Stockquote back-end service. This is done while extracting the stock volume, creating a Redis hashe map, and adding stock volume details to the Redis server. (In this example, Redis hashes are used to store different companies' stocks volume details. Since the “symbol” that is sent in the request is “WSO2”, the request is routed to the WSO2 endpoint. Once the response from the WSO2 endpoint is received, it is transformed according to the specified template and sent to the client. Then create a hashes map and insert extracted details to the Redis hashmap).
```

Comment:
```suggestion
1. Retrieve stock volume details from the Stockquote back-end service. This is done while extracting the stock volume, creating a Redis hash map, and adding stock volume details to the Redis server. (In this example, Redis hashes are used to store different companies' stocks volume details. Since the “symbol” that is sent in the request is “WSO2”, the request is routed to the WSO2 endpoint. Once the response from the WSO2 endpoint is received, it is transformed according to the specified template and sent to the client. Then create a hash map and insert extracted details to the Redis hash map).
```